camel/bots/slack/slack_app.py

```diff
@@ -0,0 +1,275 @@
+        self,
+        port: int = 3000,
+        path: str = "/slack/events",
+        host: Optional[str] = None,
+    ) -> None:
+        """Start the Slack Bolt app server to listen for incoming Slack events.
+
+        Args:
+            port (int): The port on which the server should run (default is
+                3000).
+            path (str): The endpoint path for receiving Slack events (default
+                is "/slack/events").
+            host (Optional[str]): The hostname to bind the server (default is
+                None).
+        """
+        self._app.start(port=port, path=path, host=host)
+
+    def run_in_thread(
+        self,
+        port: int = 3000,
+        path: str = "/slack/events",
+        host: Optional[str] = None,
+    ) -> None:
+        """Start the Slack Bolt app server in a separate thread.
+
+        This is useful when you want to run the server alongside other tasks
+        in the main thread.
+
+        Args:
+            port (int): The port on which the server should run (default is
+                3000).
+            path (str): The endpoint path for receiving Slack events (default
+                is "/slack/events").
+            host (Optional[str]): The hostname to bind the server (default is
+                None).
+        """
+        thread = threading.Thread(target=self.run, args=(port, path, host))
+        thread.start()
+
+    async def handle_request(
+        self, request: requests.Request
+    ) -> responses.Response:
+        """Handles incoming requests from Slack through the request handler.
+
+        Args:
+            request (Request): A Starlette request object representing the
+                incoming request.
+
+        Returns:
+            The response generated by the Slack Bolt handler.
+        """
+        return await self._handler.handle(request)
+
+    async def app_mention(
+        self,
+        context: "AsyncBoltContext",
+        client: "AsyncWebClient",
+        event: dict,
+        body: dict,
+        say: "AsyncSay",
+    ) -> None:
+        """Event handler for `app_mention` events.
+
+        This method is triggered when someone mentions the app in Slack.
+
+        Args:
+            context (AsyncBoltContext): The Slack Bolt context for the event.
+            client (AsyncWebClient): The Slack Web API client.
+            event (dict): The event data for the app mention.
+            body (dict): The full request body from Slack.
+            say (AsyncSay): A function to send a response back to the channel.
+        """
+        event_profile = SlackEventProfile(**event)
+        event_body = SlackEventBody(**body)
+
+        logger.info("on_message, context: {}".format(context))
+        logger.info("on_message, client: {}".format(client))
+        logger.info("on_message, event_profile: {}".format(event_profile))
+        logger.info("on_message, event_body: {}".format(event_body))
+        logger.info("on_message, say: {}".format(say))
```

Comment:
use `f`, same as logger below

```suggestion
        logger.info(f"on_message, context: {context}")
        logger.info(f"on_message, client: {client}")
        logger.info(f"on_message, event_profile: {event_profile}")
        logger.info(f"on_message, event_body: {event_body}")
        logger.info(f"on_message, say: {say}")
```